releasenotes/notes/fix_backend_name-e84661707058b529.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    In utils/backend_utils.py there is name() function call without checking
+    backend version. This fix adds check of backend version and get property
+    name for V2
```

A small tweak to the release note.

```suggestion
    Fixed an issue in the :class:`.QuantumInstance` class where it was assuming
    all ``AerSimulator`` backends were always :class:`.BackendV1`. This would cause
    combatibility issues with the 0.13.0 release of ``qiskit-aer`` which is starting to
    use :class:`.BackendV2` for `AerSimulator`` backends.
```

Primarily because the user facing component where the failure comes up is the `QuantumInstance` class, most users won't see the `is_statevector_backend()` utility.